authors/apps/authentication/tests/test_user_verification.py

```diff
@@ -0,0 +1,113 @@
+import time
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from authors.apps.authentication.serializers import RegistrationSerializer
+from authors.apps.authentication.views import generate_token
+
+
+class AuthenticationTests(APITestCase):
+    def setUp(self):
+        """
+        Data for the tests
+        """
+        self.data = {
+            "user": {
+                "username": "Jacob",
+                "email": "jake@jake.jake",
+                "password": "jakejake23"
+            }
+        }
+        # Set up the registration url.
+        self.reg_url = reverse('authentication:reg')
+        self.activate_url = 'authentication:activate'
```

Comment:
I can see that you haven't used reverse here, is this supposed to be this way?

Reply:
I was trying to implement `dry principle`. The reverse is added in lines where the `self.activate_url` used where I am adding `kwargs` for a token. See example in line `75-76`. I did this because the token will change in different tests